MekHQ/data/universe/eras.xml

```diff
@@ -0,0 +1,95 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+Each Era consists of the following fields:
+Code: The unique String code for the era.
+Name: The display name for the era.
+End: The end date for the era. This can be in the formats "yyyy-MM-dd", "yyyy-MM" (assuming first day of the month), and "yyyy" (assuming first day of the year). One era should have this value missing, which will put it as the final era.
+Flag: The internal flags for the base eras. You can have multiple flag fields (multiple lines each with a single flag) per custom era, which isn't used in base MekHQ, but allows one to customize things.
+
+Note that internal use of this setup is minimal at best, at least as of 0.49.3.
+-->
+<eras>
+	<era>
+		<code>PSF</code>
+		<name>Pre-Spaceflight</name>
+		<end>1950</end>
+		<flag>PRE_SPACEFLIGHT</flag>
+	</era>
+	<era>
+		<code>ESF</code>
+		<name>Early Spaceflight</name>
+		<end>2005</end>
+		<flag>EARLY_SPACEFLIGHT</flag>
+	</era>
+	<era>
+		<code>AOW</code>
+		<name>Age of War</name>
+		<end>2570</end>
+		<flag>AGE_OF_WAR</flag>
+	</era>
+	<era>
+		<code>SL</code>
+		<name>Star League</name>
+		<end>2780</end>
+		<flag>STAR_LEAGUE</flag>
+	</era>
+	<era>
+		<code>ESW</code>
+		<name>Early Succession Wars</name>
+		<end>2900</end>
+		<flag>EARLY_SUCCESSION_WARS</flag>
+	</era>
+	<era>
+		<code>LSWL</code>
+		<name>Late Succession War - LosTech</name>
+		<end>3019</end>
+		<flag>LATE_SUCCESSION_WARS_LOSTECH</flag>
+	</era>
+	<era>
+		<code>LSWR</code>
+		<name>Late Succession War - Renaissance</name>
+		<end>3049</end>
+		<flag>LATE_SUCCESSION_WARS_RENAISSANCE</flag>
+	</era>
+	<era>
+		<code>CI</code>
+		<name>Clan Invasion</name>
+		<end>3061</end>
+		<flag>CLAN_INVASION</flag>
+	</era>
+	<era>
+		<code>CW</code>
+		<name>Civil War</name>
+		<end>3067</end>
+		<flag>CIVIL_WAR</flag>
+	</era>
+	<era>
+		<code>J</code>
+		<name>Jihad</name>
+		<end>3085</end>
```

Comment:
These should be checked against the MUL. They've recently changed the dates around the Jihad.

http://www.masterunitlist.info/Era/Index

Reply:
Done, and also fixed our dates being a year out of place in a bunch of places (just copied the previous logic without checking... this actually handles it properly).